src/lib/I18n.js

```diff
@@ -35,20 +36,39 @@ const init = function (context, lang) {
   return polyglot
 }
 
+const initFormat = lang => {
+  const locales = {
+    en: require('date-fns/locale/en')
+  }
+  if (lang && lang !== 'en') {
+    try {
+      locales[lang] = require(`date-fns/locale/${lang}`)
+    } catch (e) {
+      console.warning(`The "${lang}" locale isn't supported by date-fns`)
```

Comment:
Right. We should extract this lib into cozy-ui anyway, because it's now used in 2 apps.